modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -64,6 +67,17 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
 
     typedef std::vector<int> MatShape;
 
+#if defined(CV_CXX11) || defined(CV_DOXYGEN)
+    typedef std::future<Mat> AsyncMat;
+#else
+    // Just a workaround for bindings.
+    struct AsyncMat {
+      Mat get() { return Mat(); }
+      void wait() const {}
+      size_t wait_for(size_t) const { return -1; }
```

Comment:
This definition is used to prevent `'AsyncMat’ is not a member of ‘cv::dnn’` errors in case of disabled C++11 (because shadow AsyncMat refers to it). For master branch we can use just `typedef std::future<Mat> AsyncMat;`.

Reply:
But the time in milliseconds.